console/src/app/services/grpc-auth.service.ts

```diff
@@ -249,36 +251,49 @@ export class GrpcAuthService {
         this.setActiveOrg(find);
         return Promise.resolve(find);
       } else {
-        const orgs = (await this.listMyProjectOrgs(10, 0)).resultList;
-        this.cachedOrgs.next(orgs);
-        const toFind = orgs.find((tmp) => tmp.id === id);
-        if (toFind) {
-          this.setActiveOrg(toFind);
-          return Promise.resolve(toFind);
+        const orgQuery = new OrgQuery();
+        const orgIdQuery = new OrgIDQuery();
+        orgIdQuery.setId(id);
+        orgQuery.setIdQuery(orgIdQuery);
+
+        const orgs = (await this.listMyProjectOrgs(ORG_LIMIT, 0, [orgQuery])).resultList;
```

Comment:
update this.cachedOrgs?

Reply:
same as with the other specific org quest i guess